modular_iris/modules/ordnance/maxcap_reward.dm

```diff
@@ -0,0 +1,16 @@
+/datum/techweb
+ var/maxcap_reward_gained = FALSE
+
+/obj/machinery/doppler_array
+	/// Reference to the techweb.
+	var/datum/techweb/stored_research
```

Comment:
Should clean the ref on `Destroy()`
Should perhaps also add a onqdel signal to the techweb to clean the ref if the techweb gets destroyed, tho they never get destroyed so that one's whatever